tools/validate-against-schema.py

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env python3
+import jsonschema
+import os
+import sys
+import yaml
+
+# Change current directory to this directory
+os.chdir(os.path.dirname(sys.argv[0]))
```

Comment:
We generally don't need to change directories in scripts. Instead, we can resolve paths with `os.path.join`, and friends, e.g.

```python
tools_dir = os.path.abspath(os.path.dirname(__file__))
chart_dir = os.path.join(tools_dir, os.pardir, 'jupyterhub')
templates_dir = os.path.join(tools_dir, os.pardir, 'templates')

with open(os.path.join(chart_dir, 'schema.yaml')) as f:
    ....
etc.
```

Reply:
Or you can use [pathlib](https://docs.python.org/3/library/pathlib.html) if you want to be a Fancy Person.

Reply:
Resolved by cb0ae7d9